semgrep/semgrep/semgrep_types.py

```diff
@@ -60,13 +69,57 @@ class OPERATORS:
     }
     | YAML_MUST_HAVE_KEYS
     | YAML_OPTIONAL_KEYS
+    | YAML_INTERNAL_KEYS
 )
 
+YAML_DISPLAY_VALID_RULE_KEYS = YAML_ALL_VALID_RULE_KEYS - YAML_INTERNAL_KEYS
+
 
 class InvalidRuleSchema(BaseException):
     pass
 
 
+class Span(NamedTuple):
```

Comment:
This will be confusing with Range class that is used for findings.

Reply:
moved & referenced via the namespace